### PR TITLE
Implement release process

### PR DESCRIPTION
**Pull request by Lusitaniae**
Hi again @EdSchouten ,

I think it would be nice to release this exporter the same way the official ones are built.

If you'd like to merge, you'll also need to:
    
- make an additional commit to update git, docker, quay repo urls.
- Setup Quay/Docker hub accounts.
- Setup Circle CI account, add credentias of Github, Quay and Docker to the project.

 You can see on my fork the end result.
Let me know if I can help any further.

btw, you could make an example of how to get opcache stats included as well. I've tried somethings but didin't quite get it.

### CODE REVIEW

**Comment by EdSchouten** [comment]
Hi Pedro!

First of all, these changes looks awesome! Unfortunately, we at Kumina don't have the manpower to do periodic builds and releases of our exporters. We wrote them initially because we couldn't find decent ones out there, but hope that people from the community step up to help us out.

Considering that you're already doing a pretty awesome job maintaining the Apache exporter (which we also use at Kumina), would you be interested in having write access to this repository, so you can merge this change yourself? That way development of this exporter may remain centralised. I just sent you a collaboration request.

With regards to opcache stats: we have a tiny PHP script that we use in combination with this exporter to extract those metrics. I'll try to publish it when I get to the office on Monday.

Thanks,
Ed

File: .promu.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+repository:
+    path: github.com/Lusitaniae/phpfpm_exporter
```
Comment (EdSchouten) [inline .promu.yml]: This should be `kumina` instead of `Lusitaniae`, right?

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+machine:
+  environment:
+    DOCKER_IMAGE_NAME: lusotycoon/phpfpm-exporter
+    QUAY_IMAGE_NAME: quay.io/lusitaniae/phpfpm-exporter
+    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.8-base
+    REPO_PATH: github.com/Lusitaniae/phpfpm_exporter
```
Comment (EdSchouten) [inline circle.yml]: Same here.

File: phpfpm_exporter.go [hunk 3]
```diff
@@ -16,6 +16,7 @@ package main
 import (
```
Comment (EdSchouten) [inline phpfpm_exporter.go]: Could you run this file through `gofmt`? The imports are not alphabetic right now.

**Comment by EdSchouten** [comment]
@BartVerc doesn't seem to be in the office this week, so he won't be able to review this for you. :-/

**Comment by Lusitaniae** [comment]
HI @EdSchouten ,

Much appreciated for the vote of confidence. I will be glad to help with the project.

Can you create the account for CircleCI, Quay and Docker?
I cannot add to mine, because CircleCI requires the project to be on my account.


I will run the go fmt, just a moment.

**Pull request merged**